src/components/events/EventForm.jsx

```diff
@@ -63,6 +70,12 @@ class EventForm extends React.Component {
     onModalClose();
   }
 
+  //Format of date must be DD/MM/YYYY
+  //Transforms DD/MM/YYYY to YYYY-MM-DD
+  convertDateToISOFormat = (date) => {
+    return date.substring(6, 10) + "-" + date.substring(3, 5) + "-" + date.substring(0, 2);
```

Comment:
I'd prefer we don't use the substring formatting approach if possible. Typically, date objects have a way where you can call `toString` or a similar function and pass in a string format like: `MM/DD/YYYY` which automatically parses to `01/01/1993`. Could we explore if that is possible?